src/OpenTelemetry.AutoInstrumentation/Configuration/MetricsExporter.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="MetricsExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.AutoInstrumentation.Configuration;
+
+/// <summary>
+/// Enum representing supported metrics exporters.
+/// </summary>
+public enum MetricsExporter
```

Comment:
This enum starts with `Metrics` but other types start with `Meter`. Are there OTEL naming guidelines for this? I'm unfamiliar with OTEL metrics so maybe I'm just unfamiliar with the conventions

Reply:
All metrics API based terminologies uses prefix `Meter` - https://opentelemetry.io/docs/reference/specification/metrics/api/.

Metrics Exporters starts with `Metrics` as per specification - https://opentelemetry.io/docs/reference/specification/metrics/sdk_exporters/otlp/

Reply:
Thanks for the links, that helps a lot 👍🏼